src/java/main/org/apache/zookeeper/server/ReferenceCountedACLCache.java

```diff
@@ -109,6 +109,10 @@ public synchronized void deserialize(InputArchive ia) throws IOException {
             }
             List<ACL> aclList = new ArrayList<ACL>();
             Index j = ia.startVector("acls");
+            if (j == null) {
+                LOG.error("ERROR: incorrent format of InputArchive" + ia);
```

Comment:
You don't need an ERROR in the text here or on the next line. It's already LOG'd as an error. Same for the next line - it's a RTE.

Also you need a space  "... of InputArchive" -> ".... of InputArchive " (notice space at the end. Otw the text of ia is just appended w/o the space. Also notice that ia doesn't have a toString, so I'm not sure how helpful that is.... it's fine to leave I guess.